src/index.html

```diff
@@ -22,8 +22,8 @@ <h2 class="no-margin">Recent Scores</h2>
 
     <form action="#" method="get" class="add-score d-flex">
```

Comment:
- [ ] OPTIONAL

Kindly ensure that form submission to API should be a `POST` method

Reply:
Actioned ✅

Reply:
This issue is well resolved! ✔️